pkg/operator/ceph/csi/version.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csi
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/coreos/pkg/capnslog"
+)
+
+// CephCSIVersion represents the Ceph CSI version format
+type CephCSIVersion struct {
+	Major int
+	Minor int
+}
+
+var (
+	//supportedCSIVersions are versions that rook supports
+	supportedCSIVersions = []CephCSIVersion{{1, 2}}
```

Comment:
Do we only support exactly this supported list of versions? Or should we support "at least this version"? The CSI driver should be backward compatible so the minimum supported version seems like it would be sufficient.